conseil-common/src/main/scala/tech/cryptonomic/conseil/common/tezos/TezosOptics.scala

```diff
@@ -105,7 +105,7 @@ object TezosOptics {
       */
     def extractOperationsSequence(
         block: Block
-    ): List[Either[Operation, InternalOperationResults.InternalOperationResult]] =
+    ): List[(Either[Operation, InternalOperationResults.InternalOperationResult], OperationHash)] =
```

Comment:
I agree about changing opHash to left, but this case class it's too much, because then we would want to distinguish between internal ops and regular ops results, so it would require another wrapping/unwrapping

Reply:
Isn't that a single shot operation with pattern matching on a case class?

Reply:
yes, but when we match this it would be inconsistent to leave it as a tuple elsewhere